website/src/pages/api/admin/status.ts

```diff
@@ -0,0 +1,28 @@
+import { getToken } from "next-auth/jwt";
+import { withRole } from "src/lib/auth";
+import { oasstApiClient } from "src/lib/oasst_api_client";
+
+/**
+ * Returns tasks availability, stats, and tree manager stats.
+ */
+const handler = withRole("admin", async (req, res) => {
+  const token = await getToken({ req });
+  const currentUser = {
+    id: token.sub,
+    display_name: token.name,
+    auth_method: "local",
+  };
```

Comment:
maybe you can just call `getBackendUser`? I think it should work.

Reply:
+1

Please call `getBackendUser`.  That returns the most accurate user representation.

Reply:
Done (I'm assuming you meant getBackendUserCore?)